### PR TITLE
[Release 1.0.2] Change log and Intune temporary removal

### PR DESCRIPTION
**Pull request by jianghaolu**

### CODE REVIEW

**Comment by jianghaolu** [comment]
on-demand job: [Build Status](http://azuresdkci.cloudapp.net/job/powershell-demand/415/)

File: src/ResourceManager/HDInsight/AzureRM.HDInsight.psd1 [hunk 1]
```diff
@@ -45,7 +45,7 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'
```
Comment (markcowl) [inline src/ResourceManager/HDInsight/AzureRM.HDInsight.psd1]: Please update module version to 1.0.2

File: src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1 [hunk 2]
```diff
@@ -9,7 +9,7 @@
 @{  
   
 # Version number of this module.  
-ModuleVersion = '1.1.0'
```
Comment (markcowl) [inline src/ResourceManager/KeyVault/AzureRM.KeyVault.psd1]: Should be 1.1.1, and should change the AssemblyFileVersion and AssemblyVersion in Properties\AssemblyInfo.cs to match

File: src/ResourceManager/RecoveryServices/AzureRM.RecoveryServices.psd1 [hunk 3]
```diff
@@ -45,7 +45,7 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'
```
Comment (markcowl) [inline src/ResourceManager/RecoveryServices/AzureRM.RecoveryServices.psd1]: Should be version 1.0.3

File: src/ResourceManager/SiteRecovery/AzureRM.SiteRecovery.psd1 [hunk 4]
```diff
@@ -45,7 +45,7 @@ CLRVersion='4.0'
 ProcessorArchitecture = 'None'
```
Comment (markcowl) [inline src/ResourceManager/SiteRecovery/AzureRM.SiteRecovery.psd1]: Should be version 1.1.1

**Pull request merged**